Allan-breakout-room/Evan Hameed/style.css

```diff
@@ -0,0 +1,132 @@
+*{
+
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: sans-serif;
+    color: rgb(61, 60, 60);
+}
+
+.header{
+    height: 8vh;
+    display: flex;
+    justify-content: space-evenly;
```

Comment:
instead of using ` justify-content: space-evenly` you should use ` justify-content: space-between` but that will make the header wider than enough like this 
![image](https://user-images.githubusercontent.com/55356198/80792473-385d8200-8b9d-11ea-926e-100b5608e469.png)

so the solution here is to make another div inside the header and gave that div specific width 
```
<header class="header">
   <div class=' header-wrapper'>
    <img src="./imgs/twitter_social_media_logo_icon-icons.com_76338 (1).png" alt="twitter icon">
    <div  class="links-container">
        <a href="#" class="nav-link white-link" target="_blank">LOGIN</a>
        <a href="#" class="nav-link blue-link" target="_blank" >REGISTER</a>
    </div>
  </div>
</header>  
```
```
.header-wrapper {
   width : 75% /* for example */
}